docs/dev/api_usage_examples/get_latest_import.py

```diff
@@ -0,0 +1,41 @@
+'''
+get_latest_import.py
+'''
+
+from common import AUTH_HEADER, USERNAME, construct_url
+
+import requests as req
```

Comment:
I would prefer `import requests`, renaming seems unnecessary.

Reply:
Sorry, just a habit of mine. I'm a lazy Emacs user 😝